src/sparsezoo/main.py

```diff
@@ -384,7 +389,7 @@ def main():
 
     if args.command == DOWNLOAD_COMMAND:
         LOGGER.info("Downloading files from model...")
-        model = Zoo.load_model(
+        dir_path = Zoo.download_model(
```

Comment:
Should update the variable and potentially the print statement since we're returning a model here again